README.md

```diff
@@ -27,24 +27,24 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
-    * The vscode setting 'appService.zipIgnorePattern' is changed to include build artifacts to be ignored for zipping.  These will be built on the server by running the appropriate build command.
-    * A '.deployment' file will be created in the root of the project.  This file configures SCM_DO_BUILD_DURING_DEPLOYMENT=true which enables build on deploy.
+  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click 'Yes', the following changes will be made in your project:
+    * The vscode setting `appService.zipIgnorePattern` is changed to indicate that build artifacts will not be zipped and deployed.  These will be built on the server by running the appropriate build command.
+    * A '.deployment' file will be created in the root of the project.  This file configures `SCM_DO_BUILD_DURING_DEPLOYMENT=true` enabling build on deploy.
       > NOTE: Currently only 'node' runtimes support this feature.
-  * If you select 'Never show again,' the 'appService.showBuildDuringDeployPrompt' vscode setting will be set to 'false' and you will no longer be prompted for this project.  Delete this setting or set it to 'true' to re-enable the prompt.
+  * If you select 'Never show again,' the `appService.showBuildDuringDeployPrompt` vscode setting will be set to `false` and you will no longer be prompted for this project.  Delete this setting or set it to `true` to re-enable the prompt.
 
   ### Zipdeploy Configuration Settings
 
-  * 'appService.zipGlobPattern'
-    * Uses a glob pattern to define which files to be included in the deployment. The default value is “**/*”.
+  * `appService.zipGlobPattern`
+    * Uses a glob pattern to define which files to be included in the deployment. The default value is '**/*'.
 
-  * 'appService.zipIgnorePattern'
+  * `appService.zipIgnorePattern`
     * Uses a glob pattern to define which files to be excluded from the deployment. The default value is [] which doesn’t exclude any files/folders.
 
-  * For example, you might want to exclude the “node_modules” folder from the deployment to speed up the Zip file creation and uploading. In this case, you will need the following setting:
-    * “appService.zipIgnorePattern”: ["node_modules{,/**}"]
+  * For example, you might want to exclude the 'node_modules' folder from the deployment to speed up the Zip file creation and uploading. In this case, you will need the following setting:
+    * `appService.zipIgnorePattern: ['node_modules{,/**}']`
   * And in order to have the web app run proper deployment command to restore the npm packages, you need to have the following Application Setting on your site:
-    * SCM_DO_BUILD_DURING_DEPLOYMENT=true or include the '.deployment' file that is generated.
+    * `SCM_DO_BUILD_DURING_DEPLOYMENT=true` or include the '.deployment' file that is generated.
```

Comment:
nit: It feels like "or include the '.deployment' file that is generated." should be a part of the above bullet, not this one. In other words, it could be:
```
And in order to have the web app run the proper deployment command to install the npm packages, you need to have the following Application Setting on your site or in a '.deployment' file at the root of your repo:
```